gometalinter/def.bzl

```diff
@@ -55,6 +57,10 @@ _gometalinter = rule(
             mandatory = True,
             doc = "Go import path of this project i.e. where in GOPATH you would put it. E.g. github.com/atlassian/bazel-tools",
         ),
+        "fast": attr.bool(
```

Comment:
This can be set via configuration file https://github.com/alecthomas/gometalinter/blob/master/config.go, along with many other directives.

Reply:
Then you need two configuration files. In our use case this is orthogonal to the set of linters we want to run and their configuration.